after gcc linking has changed, all the libraries must be explicitely specified 
This patch avoids this linking error:

| make[1]: Entering directory `/disk0/pokybuild/build1/tmp/work/i586-poky-linux/blktrace-1.0.1+git0+1e09f6e9012826fca69fa07222b7bc53c3e629ee-r1/git/btreplay'^M
| ccache i586-poky-linux-gcc -march=i586 --sysroot=/disk0/pokybuild/build1/tmp/sysroots/i586-poky-linux -fexpensive-optimizations -fomit-frame-pointer -frename-registers -O2 -ggdb -feliminate-unused-debug-types -I. -I.. -I../btt -D_GNU_SOURCE -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -UCOUNT_IOS -UDEBUG -DNDEBUG -o btreplay btreplay.o -laio -lrt^M
| /disk0/pokybuild/build1/tmp/sysroots/x86_64-linux/usr/libexec/i586-poky-linux/gcc/i586-poky-linux/4.5.1/ld: >: invalid DSO for symbol `pthread_join@@GLIBC_2.0' definition^M
| /disk0/pokybuild/build1/tmp/sysroots/i586-poky-linux/lib/libpthread.so.0: could not read symbols: Bad value^M
| collect2: ld returned 1 exit status^M
| make[1]: *** [btreplay] Error 1

Nitin A Kamble <nitin.a.kamble@intel.com>
Date: 2011/01/11

Index: git/btreplay/Makefile
===================================================================
--- git.orig/btreplay/Makefile
+++ git/btreplay/Makefile
@@ -11,7 +11,7 @@ XCFLAGS	= -D_GNU_SOURCE -D_LARGEFILE_SOU
 override CFLAGS += $(INCS) $(XCFLAGS) $(OCFLAGS)
 
 PROGS	= btrecord btreplay
-LIBS	= -laio -lrt
+LIBS	= -laio -lrt -lpthread
 
 all: depend $(PROGS)
 
